jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -220,6 +220,17 @@ YUI.add('deployment-flow', function() {
     },
 
     /**
+      Handle clearing the chosen cloud.
+
+      @method _changeCloud
+    */
+    _changeCloud: function() {
```

Comment:
As this function only clears the selected cloud. Maybe it should be named as such. For example `_clearCloud`.